docs/userguide/datetime/calendar/examples.md

```diff
@@ -11,12 +11,12 @@ These C++, C , and Java examples get a Calendar based on the default time zone
 and add days to a date.
 
 **C++**
-
+````C++
 UErrorCode status = U_ZERO_ERROR;
 GregorianCalendar\* gc = new GregorianCalendar(status);
 if (U_FAILURE(status)) {
-puts("Couldn't create GregorianCalendar");
-return;
+  puts("Couldn't create GregorianCalendar");
```

Comment:
please fix code indentation to 4 spaces per level (multiple code snippets)

Reply:
Done